modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionServiceImpl.java

```diff
@@ -46,15 +47,16 @@
 
 	@Override
 	public ListTypeDefinition addListTypeDefinition(
-			String externalReferenceCode, Map<Locale, String> nameMap)
+			String externalReferenceCode, List<ListTypeEntry> listTypeEntries,
```

Comment:
I think we should order the parameters by following the order from service.xml. Since ListTypeDefinition does not contain the listTypeEntries, so I believe that this parameter should come at last.